### PR TITLE
[DOCS] Minor classref updates.

### PR DESCRIPTION
**Pull request by cbscribe**
* Correct verb tense in Array.xml
* Description and code example for SceneTreeTimer and SceneTree.create_timer()

### CODE REVIEW

File: doc/classes/Array.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		Generic array datatype.
 	</brief_description>
 	<description>
-		Generic array, contains several elements of any type, accessible by a numerical index starting at 0. Negative indices can be used to count from the back, like in Python (-1 is the last element, -2 the second to last, etc.). Example:
+		Generic array which may contain several elements of any type, accessible by a numerical index starting at 0. Negative indices can be used to count from the back, like in Python (-1 is the last element, -2 the second to last, etc.). Example:
```
Comment (aaronfranke) [inline doc/classes/Array.xml]: "which can contain" would probably be better.

File: doc/classes/Array.xml [hunk 2]
```diff
@@ -144,21 +144,21 @@
...
-				If [code]deep[/code] is [code]true[/code], a deep copy is be performed: all nested arrays and dictionaries are duplicated and will not be shared with the original array. If [code]false[/code], a shallow copy is made and references to the original nested arrays and dictionaries are kept, so that modifying a sub-array or dictionary in the copy will also impact those referenced in the source array.
+				If [code]deep[/code] is [code]true[/code], a deep copy is performed: all nested arrays and dictionaries are duplicated and will not be shared with the original array. If [code]false[/code], a shallow copy is made and references to the original nested arrays and dictionaries are kept, so that modifying a sub-array or dictionary in the copy will also impact those referenced in the source array.
 			</description>
 		</method>
 		<method name="empty">
 			<return type="bool">
 			</return>
 			<description>
-				Return true if the array is empty (size==0).
+				Returns [code]true[/code] if the array is empty (size==0).
```
Comment (aaronfranke) [inline doc/classes/Array.xml]: Should "(size==0)" also have "[code]"?

**Review by aaronfranke — APPROVED** [review]

File: doc/classes/Array.xml (hunk 1 above)
Comment (cbscribe) [reply doc/classes/Array.xml]: Either works for me.

File: doc/classes/Array.xml (hunk 2 above)
Comment (cbscribe) [reply doc/classes/Array.xml]: I had thought about just removing that parenthetical. "empty" is pretty self-explanatory.

File: doc/classes/Array.xml [hunk 3]
```diff
@@ -305,7 +305,7 @@
 		</method>
 		<method name="sort">
 			<description>
-				Sort the array using natural order.
+				Sorts the array using natural order.
```
Comment (merumelu) [inline doc/classes/Array.xml]: This description is misleading. Sorting in [natural order](https://en.wikipedia.org/wiki/Natural_sort_order) means sorting strings in a human-friendly way but that's not what this method does.
```gdscript
var a = ['22 days', '2 days', '17 days']
a.sort()
print(a) # => [17 days, 2 days, 22 days] but with natural order would be [2 days, 17 days, 22 days]
```

Comment (cbscribe) [reply doc/classes/Array.xml]: That's a good catch! I don't think I've ever needed to sort a string array like that. What do we think is a better description here: "Sorts the array. Note: strings are sorted in alphabetical, not natural order."

Comment (aaronfranke) [reply doc/classes/Array.xml]: I believe the technical term is lexicographical order.

Comment (cbscribe) [reply doc/classes/Array.xml]: "alphabetical" is a more common synonym, so I think we'll stick with that.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**